$NetBSD: patch-configure.ac,v 1.1 2021/01/23 13:12:32 mef Exp $

with autoconf 2.70:
configure.ac:43: error: AC_CONFIG_MACRO_DIR can only be used once

--- configure.ac.orig	2020-11-06 10:50:51.000000000 +0900
+++ configure.ac	2021-01-23 21:48:16.150573447 +0900
@@ -40,7 +40,7 @@ AC_INIT([LTFS], [2.4.3.0 (10450)], IBM c
 AC_CONFIG_SRCDIR([src/main.c])
 AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIRS([m4])
-AC_CONFIG_MACRO_DIR([m4])
+dnl AC_CONFIG_MACRO_DIR([m4])
 AC_CANONICAL_HOST
 AC_CANONICAL_BUILD
 AC_CANONICAL_TARGET
